minisom.py

```diff
@@ -434,14 +435,19 @@ def train_batch(self, data, num_iteration, verbose=False):
         """
         self.train(data, num_iteration, random_order=False, verbose=verbose)
 
-    def distance_map(self):
+    def distance_map(self, neighbour_average=False):
```

Comment:
I would call the new parameter `scaling` and have it as a string. This way the user can scale by the sum using the method like this

```
distance_map(scaling='sum')
```

or scale by the average using it like this

```
distance_map(scaling='mean')
```